.github/workflows/ci.yml

```diff
@@ -182,6 +194,95 @@ jobs:
           path: generator.diff
           if-no-files-found: ignore
 
+  kitchensink-go:
+    runs-on: ubuntu-latest
+    needs: build-lint-test-go
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Install Tools
+        run: |
+          ./scripts/install-go.sh
+      - name: Test Kitchensink Go
+        run: |
+          SDK=go go test ./loadgen -run TestKitchensink -v
+
+  kitchensink-java:
+    runs-on: ubuntu-latest
+    needs: build-lint-test-java
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Install mise
+        uses: jdx/mise-action@v2.4.4
+        with:
+          version: ${{ env.MISE_VERSION }}
+      - name: Install Tools
+        run: |
+          ./scripts/install-java.sh
+          ./scripts/install-go.sh
+      - name: Test Kitchensink Java
+        run: |
+          SDK=java go test ./loadgen -run TestKitchensink -v
+
+  kitchensink-python:
+    runs-on: ubuntu-latest
+    needs: build-lint-test-python
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+      - name: Install mise
+        uses: jdx/mise-action@v2.4.4
+        with:
+          version: ${{ env.MISE_VERSION }}
+      - name: Install Tools
+        run: |
+          ./scripts/install-python.sh
+          ./scripts/install-go.sh
+      - name: Test Kitchensink Python
+        run: |
+          SDK=python go test ./loadgen -run TestKitchensink -v
+
+  kitchensink-typescript:
+    runs-on: ubuntu-latest
+    needs: build-lint-test-typescript
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: Install mise
+        uses: jdx/mise-action@v2.4.4
+        with:
+          version: ${{ env.MISE_VERSION }}
+      - name: Install Tools
+        run: |
+          ./scripts/install-node.sh
+          ./scripts/install-go.sh
+      - name: Test Kitchensink TypeScript
+        run: |
+          SDK=typescript go test ./loadgen -run TestKitchensink -v
+
+  kitchensink-dotnet:
+    runs-on: ubuntu-latest
+    needs: build-lint-test-dotnet
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: Install mise
+        uses: jdx/mise-action@v2.4.4
+        with:
+          version: ${{ env.MISE_VERSION }}
+      - name: Install Tools
+        run: |
+          ./scripts/install-dotnet.sh
+          ./scripts/install-go.sh
+      - name: Test Kitchensink .NET
+        run: |
+          SDK=dotnet go test ./loadgen -run TestKitchensink -v
+
```

Comment:
Separate jobs for two reasons:

1. the workers just log to stdout and it can be difficult to read the logs
2. parallelism